tests/html5skin-test.js

```diff
@@ -1180,4 +1181,28 @@ describe('Controller', function() {
       expect(controller.state.pluginsElement.hasClass('oo-overlay-showing')).toEqual(true);
     });
   });
+
+  describe('Config settings', function() {
+    var spyPublish;
+
+    beforeEach(function() {
+      spyPublish = sinon.spy(OO.mb, 'publish');
+    });
+
+    afterEach(function() {
+      spyPublish.restore()
+    });
+
+    it('test that the chosen ui language is sent on the message bus', function() {
+      controller.state.elementId = elementId;
```

Comment:
Can we add this to the beforeEach block? This way if someone else adds new unit tests, they are less likely to face the same issue you did.